### PR TITLE
Update Pokémon R/B options

### PR DESCRIPTION
**Pull request by Alchav**

### CODE REVIEW

File: games/Pokemon Red and Blue.yaml [hunk 1]
```diff
@@ -88,9 +88,30 @@ Pokemon Red and Blue:
...
+    false: 3
+  all_pokemon_seen:
+    true: 1
+    false: 1
   trainersanity:
     true: 1
     false: 3
+  randomize_rock_tunnel:
+    true: 1
+    false: 1
```
Comment (t3hf1gm3nt) [inline games/Pokemon Red and Blue.yaml:105]: so could you explain a bit better what the warning is about with this particular option? like is it still possible for things to be unreachable in its current state? if so im not sure this should be an option for the filler yaml until these issues are resolved

**Comment by t3hf1gm3nt** [comment]
also congrats on getting PR 69. nice

File: games/Pokemon Red and Blue.yaml (hunk 1 above)
Comment (Alchav) [reply games/Pokemon Red and Blue.yaml:105]: In the sense that it's possible that I made a mistake that hasn't been yet discovered

Comment (t3hf1gm3nt) [reply games/Pokemon Red and Blue.yaml:105]: For safety's sake until you are absolutely certain there are no issues with this option, I'm gonna request this option always being off for the fillers.
Once that's done, everything else looks fine on this and I'll merge it

**Pull request merged**